recipes/mailio/all/conanfile.py

```diff
@@ -70,9 +70,21 @@ def validate(self):
         except KeyError:
             self.output.warn(f"{self.ref} has no support for the current compiler. Please consider adding it.")
 
+    def _cmake_new_enough(self, required_version):
```

Comment:
This should not be required with the improved cmake recipe using portable binaries

Reply:
It's orthogonal. As a user I don't want these cmake build requirements if I have a recent cmake version.